client/packages/requisitions/src/RnRForms/api/hooks/useRnRFormContext.ts

```diff
@@ -149,11 +163,21 @@ export const useCachedRnRDraftLine = (id: string) => {
     const baseLine = state.baseLines[id];
     if (!baseLine) return undefined;
 
-    return previousIteration == (state.draftLineIteration[id] ?? 0)
-      ? prev.current
-      : (prev.current = {
-          line: { ...baseLine, ...(state.draftLines[id] || {}) },
+    const line = { ...baseLine, ...(state.draftLines[id] || {}) };
+
+    const highlight =
```

Comment:
I wasn't happy with this one for some reason, mainly cause i went all out on the state.draftLineIteration, anyways downstream it's changed a bit